src/paperless/settings.py

```diff
@@ -1045,8 +1045,8 @@ def default_threads_per_worker(task_workers) -> int:
 if TIKA_ENABLED:
     INSTALLED_APPS.append("paperless_tika.apps.PaperlessTikaConfig")
 
-AUDIT_LOG_ENABLED = __get_boolean("PAPERLESS_AUDIT_LOG_ENABLED", "NO")
```

Comment:
I feel like we shouldn't change the name of the setting, but instead just negate it internally

Reply:
I feel like it would be a breaking change otherwise, since someone might have it set and think it is doing something, when it is in fact no longer checked

Reply:
Yea good point, no reason not to do just flip it